api/src/main/java/io/opentelemetry/metrics/Measure.java

```diff
@@ -37,6 +51,9 @@
      * @param absolute {@code true} only positive values are expected.
      * @return this.
      */
-    B setAbsolute(boolean absolute);
+    Builder setAbsolute(boolean absolute);
+
+    @Override
+    Measure<?> build();
```

Comment:
same comment about wildcard return values

Reply:
This is not directly used by any user. They will use LongMeasure or DoubleMeasure, it is here to make sure we always override build.

Reply:
Let's go with this for now. It feels pretty kludgy; I'll try to come up with something a bit cleaner in the future.